Data-tranformations/fairgenomes/analyses.yarrrml.yaml

```diff
@@ -3,24 +3,55 @@ prefixes:
   NCIT: http://purl.obolibrary.org/obo/NCIT_
   OMIABIS: http://purl.obolibrary.org/obo/OMIABIS_
   OBI: http://purl.obolibrary.org/obo/OBI_
-  ARF: http://purl.allotrope.org/ontologies/result# #this is new and it ends with a #
-  EDAM:http://edamontology.org/ 
+  ARF: "http://purl.allotrope.org/ontologies/result#"
+  EDAM: http://edamontology.org/ 
   FIX: http://purl.obolibrary.org/obo/FIX_
   GENEPIO: http://purl.obolibrary.org/obo/GENEPIO_
+  fgs: https://w3id.org/fair-genomes/ontology/ 
 
-mappings:
+
+mappings: # mapping for analysis
+  fairgenomes:
+    sources:
+      - [analyses.csv~csv]
+    s: ega.analyses:$(accession_id)
+    po:
+      - [a, EDAM:operation_2945] # "Analysis"
+      - [a, fgs:Analysis]
+      - [OMIABIS:0000006, ega.analyses:$(accession_id)~iri] # I am unsure about this. However I feel that this OMIABIS:0000006 is wrong
```

Comment:
I have not really understood this command, to be honest. I really believe that, in this case, the OMIABIS ... is wrong. But how should I think in order to find the appropriate ontology in this case?